cns-hook

```diff
@@ -191,6 +212,29 @@ function unchanged_cert {
 		fi
 		echo "OK: adminui certificate up to date" >&2
 		;;
+	sdcsso)
+                vmuuid="$(vmadm lookup alias=sdcsso)"
+                local target="/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/cert.pem"
+                local target_key="/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/key.pem"
+                local restart=false
+
+                if ! diff "${certdir}/fullchain.pem" "${target}" >/dev/null; then
+                        cp "${certdir}/fullchain.pem" "/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/cert.pem"
+                        restart=true
+                fi
+
+                if ! diff "${keyfile}" "${target_key}" >/dev/null; then
+                        cp "${certdir}/privkey.pem" "/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/key.pem"
+                        restart=true
+                fi
+
+                if ${restart} == true; then
+                        zlogin "${vmuuid}" svcadm restart sdcsso
+                        echo "OK: sdcsso certificate/key deployed, and sdcsso restarted" >&2
+                else
+                        echo "OK: sdcsso certificate/key up to date" >&2
+                fi
+		;;
```

Comment:
Indentation throughout this whole change here doesn't match what's already in the file (and doesn't match the ;; line either)